spec/controllers/advice_controller_spec.rb

```diff
@@ -22,4 +23,134 @@
       end
     end
   end
+
+  describe '#invalid_advice_status' do
+    context "when invalid_advice_status is called with question advice score > max score of questionnaire" do
+      #max score of advice = 3 (!=2)
+      let(:questionnaire) do
+        build(:questionnaire, id: 1, min_question_score: 1,
+          questions: [build(:question, id: 1, weight: 2, question_advices: [build(:question_advice, id:1, score: 1, question_id: 1, advice: "Advice1"), build(:question_advice, id:2, score: 3, question_id: 1, advice: "Advice2")])], max_question_score: 2)
```

Comment:
Can you break down these long lines of code into smaller lines, possibly break down the nested objects into separate variables?